threatx/info.json

```diff
@@ -0,0 +1,353 @@
+{
+  "name": "threatx",
+  "label": "ThreatX",
+  "description": "Use the ThreatX integration to enrich intel and automate enforcement actions on the ThreatX Next Gen WAF.",
+  "publisher": "Fortinet",
+  "cs_approved": true,
+  "cs_compatible": true,
+  "version": "1.0.0",
+  "category": "Network Security",
+  "icon_small_name": "small.png",
+  "icon_large_name": "large.png",
+  "help_online": "",
+  "configuration": {
+    "fields": [
+      {
+        "title": "Server URL",
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "type": "text",
+        "name": "server_url",
+        "description": "URL of the ThreatX server to which you will connect and perform the automated operations."
+      },
+      {
+        "title": "API Key",
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "type": "password",
+        "name": "api_key",
+        "description": "The api key for the ThreatX server to which you will connect and perform the automated operations."
+      },
+      {
+        "title": "Customer Name",
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "type": "text",
+        "name": "customer_name",
+        "description": "The customer name for the ThreatX server to which you will connect and perform the automated operations."
+      },
+      {
+        "title": "Verify SSL",
+        "required": false,
+        "editable": true,
+        "visible": true,
+        "type": "checkbox",
+        "name": "verify_ssl",
+        "value": true,
+        "description": "Specifies whether the SSL certificate for the server is to be verified or not. By default, this option is set as True."
+      }
+    ]
+  },
+  "operations": [
+    {
+      "operation": "block_ip",
+      "title": "Block IP Address",
+      "description": "Add IP Address entry to blocklist in Threat X.",
+      "category": "investigation",
+      "annotation": "block_ip",
+      "output_schema": {"Ok": ""},
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "IP Address",
+          "required": false,
```

Comment:
This parameter is un-required? Can you please check.
Also, check for other parameters of this operation.